docs/02-app/02-api-reference/04-functions/generate-metadata.mdx

```diff
@@ -521,6 +521,9 @@ export const metadata = {
 <meta property="og:image:width" content="1800" />
 <meta property="og:image:height" content="1600" />
 <meta property="og:image:alt" content="My custom alt" />
+<meta property="og:video:url" content="https://nextjs.org/video.mp4" />
```

Comment:
`og:video:url` is not a valid property; https://ogp.me/#structured

Unlike `og:image`, which does have a `:url` variant, video does not

Reply:
Good catch, it should be `og:video`

Reply:
As audio & video do not have a `:url` variant, should we make this consistent for image?

Reply:
Nice!

In addition, `og:image` examples still use `og:image:url` in docs.
We will update the below code and docs.
https://github.com/vercel/next.js/blob/2a87b33ff344c6ac4734498d9d4611d10c3aab7a/packages/next/src/lib/metadata/generate/meta.tsx#L56-L61

Reply:
https://github.com/vercel/next.js/pull/69959, ahh already did :P